app/styles/ui/_tab-bar.scss

```diff
@@ -108,4 +117,45 @@
       font-weight: var(--font-weight-semibold);
     }
   }
+
+  &.vertical &-item {
+    // Reset styles from global buttons
+    cursor: pointer;
+    border: none;
+    box-shadow: none;
+    border-radius: var(--border-radius);
+    color: var(--text-color);
+    background: var(--background-color);
+    font-family: var(--font-family-sans-serif);
+    font-size: var(--font-size);
```

Comment:
looks like these are copied from `&.switch &-item`, perhaps they could be shared between the two classes somehow? maybe with a shared base class or something?